blog/_posts/2020-08-12-building-root-with-distcc.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Building ROOT fast - distributed compilation with distcc"
+layout: archive
+author: Stefan Wunsch
+---
+
+Hello there, here is Stefan with a short guide on how you can compile ROOT distributed with distcc!
+
+ROOT is a huge project with more than 24 thousand files and above 4.6 million lines of code (counted with [`cloc`](https://github.com/AlDanial/cloc)). Consequently, a full rebuild from scratch takes a while! To give some numbers:
```

Comment:
```suggestion
ROOT is a huge project with more than 24 thousand files and 4.6 million lines of code (counted with [`cloc`](https://github.com/AlDanial/cloc)). That's because it comes with a full compiler, which [interprets C++ code](https://root.cern/primer/#learn-c-at-the-root-prompt). Consequently, a full rebuild from scratch takes a while! To give some numbers:
```

Reply:
Already took the numbers without LLVM!